structs.h

```diff
@@ -861,7 +861,7 @@ struct PlayerStruct
 	int _pVar6;
 	int _pVar7;
 	int _pVar8;
-	unsigned char _pLvlVisited[17];
+	unsigned char _pLvlVisited[17]; // NUMLEVELS

```

Comment:
Why only a comment here?

Reply:
I wanted to wait until `_pSLvlVisited[10]` is figured out, since there's only 5 set levels but the size is 10. Comparing this to Hellfire would let us know for sure.